ModernisedDarts.postman_collection.json

```diff
@@ -1940,6 +2006,64 @@
 					]
 				}
 			]
+		},
+		{
+			"name": "NodeRegistration",
```

Comment:
Should this folder be `Devices` in keep with the current tag in the OAS?

Reply:
I used the name of the yaml file but can change it if you prefer?

Reply:
I guess so, but up to you :) - annoying history with the good old `node_register`.
But the path is also currently `/register-devices` so no mention of node